lib/modules/platform/bitbucket/index.ts

```diff
@@ -1065,6 +1070,14 @@ export async function updatePr({
       `/2.0/repositories/${config.repository}/pullrequests/${prNo}/decline`,
     );
   }
+
+  // update pr cache
+  await BitbucketPrCache.addPr(
```

Comment:
The add function also does updating too?

Reply:
Yes.

https://github.com/renovatebot/renovate/blob/aabf1638d1a39fa675d2eecc7a23076b378c1f28/lib/modules/platform/bitbucket/pr-cache.ts#L77-L90

Reply:
Please rename it to setPr()